app/services/ucas_matching/matching_data_export.rb

```diff
@@ -0,0 +1,109 @@
+module UCASMatching
+  class MatchingDataExport
+    HEADER_NAMES = {
+      apply_candidate_id: 'Apply candidate ID',
+      first_name: 'First Name',
```

Comment:
Should we lower-case `name` to be consistent with other two-word headers?

Reply:
This is not a change I made, this is the file format that's agreed with UCAS.